thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -241,7 +241,6 @@ export default Component.extend({
 
       return filterPrefix(displayableUrns, 'frontend:metric:current:')
         .map(urn => [toMetricLabel(toMetricUrn(urn), entities), urn])
-        .sort()
```

Comment:
is this intentional? they should be sorted

Reply:
When sorted (by id) the metrics loaded in graph don't match the order in the legend.